devito/dle/parallelizer.py

```diff
@@ -61,12 +62,17 @@ def default_value(cls):
         return nhyperthreads()
 
     def __new__(cls, **kwargs):
-        name = kwargs.get('name', NThreadsNested.name)
+        name = kwargs.get('name', cls.name)
         value = NThreadsNested.default_value()
         return super(NThreadsNested, cls).__new__(cls, name=name, dtype=np.int32,
                                                   value=value)
 
 
+class NThreadsNonaffine(NThreads):
+
+    name = 'nthreads2'
```

Comment:
Maybe rename these according to class, like `nthreadAff` and so on?

Reply:
I thought about that, but the generated code looks a bit ugly that way... I'm paranoid, aren't I? 

I thought it's just three names and they'll never change, tied deterministically to specific classes of loops, so it shouldn't be too bad ?

However, I'm willing to reconsider if you feel like this is gonna be a mental burnout for users

Reply:
I don't know with a 3 letter thing like `Aff, Std, ...` I think it's still fairly readable and makes it a bit better for the readability of the C code.

Reply:
OK. I've just finished it. Eventually, I picked `nthreads` (basically "all classic stencil loops"), `nthreads_nested`, `nthreads_nonaffine`.